### DISCUSSION TITLE
Configuration of search

### DISCUSSION THREAD

**Discussion by SharkSyl** [squidfunk/mkdocs-material#4764] [Q&A] [answered]
```python
theme:
  name: material
  language: zh
  logo: server.ico
  palette:
    primary: light blue
  features:
    - search.suggest
    - search.highlight
    - navigation.instant
    - navigation.tracking
    
plugins:
  - search:
      prebuild_index: true 
      separator: '[\#\s\u200b\-\.]'
      indexing: 'full'
```

1.  ```search.separator```  is invalid?
According to ```search.separator``` configuration, the search condition ```base#api``` is invalid and ```base api``` is valid.
2.   ```search.prebuild_index```   long initialization search time
The index was preconstructed, but the initial search cost nearly two minutes.
```/search/search_index.json``` file size is 116MB, is this related?

**Comment by squidfunk** [comment, marked as answer]
Note that [`prebuild_index`](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/#+search.prebuild_index) is not supported by Material for MkDocs since version 8 (see link). Additionally, I invite you to try version 9 beta (tracked in #4714), which contains the search rewrite that is [faster and yields smaller search indexes](https://squidfunk.github.io/mkdocs-material/blog/2021/09/13/search-better-faster-smaller/).

If that doesn't help, please provide a self-contained reproducible example.

**Reply by SharkSyl** [reply]
I tried using version 9.0.0b4 and the search initialization time was 9 seconds, which I can say is a big improvement compared to a few minutes, and search.separator is working properly.

**Reply by SharkSyl** [reply]
thanks
